wolfcrypt/test/test.c

```diff
@@ -24462,67 +24455,26 @@ WOLFSSL_TEST_SUBROUTINE int ecc_encrypt_test(void)
 #endif
     word32  outSz2   = sizeof(out2);
     word32  plainSz2 = sizeof(plain2);
-
-#ifndef HAVE_FIPS
-    ret = wc_InitRng_ex(&rng, HEAP_HINT, devId);
+#ifdef WOLFSSL_SMALL_STACK
+    ecc_key *tmpKey = (ecc_key *)XMALLOC(sizeof *userB, HEAP_HINT,
```

Comment:
Note mismatch `userB` and `tmpKey` for destination and sizeof... Both are `ecc_key`, but might cause future issue.

Reply:
fixed